definitions/ext-datadog_system/definition.yml

```diff
@@ -0,0 +1,18 @@
+domain: EXT
+type: DATADOG_SYSTEM
```

Comment:
I thought this was meant to be MYSQL? (`type` and the directory)

Reply:
Oh ignore this, I didn't realize it's a separate entity

Reply:
i'm wondering if system is a more broad definition that could include other types than host? That said +1 to using EXT-HOST

Reply:
A question on this: There is already an `ext-host` directory and definition, what should be the approach for changing this over to  `ext-host`?

Reply:
Bumping this question and giving a bit more background: these entities are to support specifically Datadog entries coming in from an internal service hooked up to a datadog agent, with a datadog. prefix we are appending upon intake.  Should they still be switched over to `EXT-HOST` and `EXT-MYSQL` in that instance, or wouldn't that make them too generic?

Reply:
@AustinNester we will be able to determine source of telemetry via the `instrumentation.provider=datadog` tag that will be pinned to all data points